apps/contrib/templates/a4_candy_contrib/includes/form_field.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n widget_tweaks %}
 
 <div class="form-group">
-    <label for="{{ field.id_for_label }}">
+    <label for="{{ field.id_for_label }}" {% if tabindex is not None %}tabindex="{{ tabindex }}"{% endif %}>
```

Comment:
@fuzzylogic2000 is this ok to do? it works but not sure if this is the place for logic, sorry will try get better with the django stuff!

Reply:
Yes, that is a very good idea! Should have thought of that. :)
We could call the variable legal_tabindex to make more clear where it goes, but we can also change that when we need to add a tabindex elsewhere.
Should I do the commit squashing?

Reply:
@fuzzylogic2000 yes please! thank you!